### ISSUE TITLE
stateful set is showing progressing 

### ISSUE THREAD

**Issue by wanghong230** [argoproj/argo-cd#668]
<img width="1130" alt="screen shot 2018-10-02 at 12 04 29 am" src="https://user-images.githubusercontent.com/2680725/46333859-cb57c400-c5d6-11e8-830b-d4ae045a6929.png">

**Comment by alexmt** [comment]
@wanghong230 we are using the same logic as `kubectl rollout status` . Can you please run `kubectl rollout status statefulsets redis-cluster` and post output?

**Comment by wanghong230** [comment]
```
MTVL1679decd5:iksm hwang8$ kubectl rollout status statefulsets redis-cluster
Waiting for partitioned roll out to finish: 0 out of 6 new pods have been updated...
^CMTVL1679decd5:iksm hwang8$ kubectl get statefulsets redis-cluster
NAME            DESIRED   CURRENT   AGE
redis-cluster   6         6         1d
```

Might be a kubenetes bug

**Comment by alexmt** [comment]
You are right: https://github.com/kubernetes/kubernetes/issues/68573 .  Fix will be available in 1.11 . Closing ticket since no changes are required from argocd side

**Comment by jessesuen** [comment]
FTR, here is the fix:
https://github.com/kubernetes/kubernetes/issues/52653. 

Cherry-picked to v1.11:
https://github.com/kubernetes/kubernetes/pull/67570
